lib/ansible/modules/storage/glusterfs/gluster_peer.py

```diff
@@ -0,0 +1,160 @@
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: gluster_peer
+short_description: Attach/Detach peers to/from the cluster
+description:
+  - Create or diminish a GlusterFS trusted storage pool. A set of nodes can be
+    added into an existing trusted storage pool or a new storage pool can be
+    formed. Or, nodes can be removed from an existing trusted storage pool.
+version_added: "2.6"
+author: Sachidananda Urs (@sac)
+options:
+    state:
+       choices: ["present", "absent"]
+       description:
+          - Determines whether the nodes should be attached to the pool or
+            removed from the pool. If the state is present, nodes will be
+            attached to the pool. If state is absent, nodes will be detached
+            from the pool.
+       required: true
+    nodes:
+       description:
+          - List of nodes that have to be probed into the pool.
+       required: true
+    force:
+       choices: ["yes", "no"]
+       default: "no"
+       description:
+          - Applicable only while removing the nodes from the pool. gluster
+            will refuse to detach a node from the pool if any one of the node
+            is down, in such cases force can be used.
+requirements:
+  - PyYAML
+  - GlusterFS > 3.2
+'''
+
+EXAMPLES = '''
+- name: Create a trusted storage pool
+  gluster_pool:
+        state: present
+        nodes:
+             - 10.0.1.5
+             - 10.0.1.10
+
+- name: Delete a node from the trusted storage pool
+  gluster_pool:
+         state: absent
+         nodes:
+              - 10.0.1.10
+
+- name: Delete a node from the trusted storage pool by force
+  gluster_pool:
+         state: absent
+         nodes:
+              - 10.0.0.1
+         force: yes
+'''
+
+RETURN = '''
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ast import literal_eval
+
+
+class Peer(object):
+    def __init__(self, module):
+        self.module = module
+        self.state = self._validated_params('state')
+
+    def _validated_params(self, opt):
+        if self.module.params[opt] is None:
+            msg = "Please provide %s option in the playbook!" % opt
+            self.module.fail_json(rc=4, msg=msg)
+        return self.module.params[opt]
+
+    def get_nodes(self):
+        nodes = literal_eval(self._validated_params('nodes'))
+        if nodes[0] is None:
+            # Found a list with None as its first element
+            self.module.exit_json(rc=1, msg="No nodes found, provide at least "
+                                  "one node.")
+        return nodes
+
+    def gluster_peer_ops(self):
+        nodes = self.get_nodes()
+        force = 'force' if self.module.params.get('force') == 'yes' else ''
```

Comment:
Since `force` should be a boolean (see comment below), it will be `force = 'force' if self.module.params.get('force') else ''`.

Reply:
Done.